### DISCUSSION TITLE
点击翻译书籍后，会滚动显示一些信息，但是太快了看不清。这些信息如何查看？

### DISCUSSION THREAD

**Discussion by GoTop** [bookfere/Ebook-Translator-Calibre-Plugin#270] [Q&A]
ebook-translator_20240329182722版在点击翻译书籍后，会滚动显示一些信息，但是太快了

这些信息如何查看？

**Comment by bookfere** [comment]
用鼠标按住滚动条向上拖一下就不会自动滚动了，拖到最底部恢复自动滚动。

**Comment by GoTop** [comment]
点击Calibre软件的菜单栏上的“翻译书籍”图标后，弹出的分割书籍的界面上，右边出现的信息，好像没办法安装你说的方法让它停止滚动呢
